internal/cssc/cssc.go

```diff
@@ -192,6 +192,7 @@ func ApplyFilterAndGetFilteredList(ctx context.Context, acrClient api.AcrCLIClie
 		patchTagRegex = floatingTagRegex
 	}
 
+	uniqueFilteredRepos := make(map[string]bool)
```

Comment:
It really doesn't matter much for this scenario given the size but its typical to use a map[string]struct{} to represent sets in golang because struct{} uses no memory